src/styles/card.css

```diff
@@ -0,0 +1,89 @@
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  --primaryColor: #00acdc;
+  --primaryTextColor: #060b35;
+  --secondaryTextColor: #616070;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.big-text {
+  font-weight: 700;
+  font-size: 1.6em;
+  line-height: 18px;
+}
+
+.small-text {
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 14px;
+}
+
+.card__title {
+  padding-top: 40px;
+  color: var(--primaryTextColor);
+  font-weight: 500;
+  font-size: 1.2em;
+}
+
+.card__subtitle {
+  color: var(--secondaryTextColor);
+  padding-top: 4px;
+  width: 100%;
+}
+
+.reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding-top: 16px;
+}
+
+.reviews__text {
+  color: var(--primaryTextColor);
+  padding-left: 17px;
+}
+
+.card__payment {
+  padding: 24px 0 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__payment-subtitle {
+  color: var(--secondaryTextColor);
+  font-size: 1.2em;
+}
+
+.card__btn {
+  font-size: 1.4em;
+  line-height: 16px;
+  text-decoration: none;
+  padding: 11px 69px 11px 70px;
```

Comment:
let's make the button spread all over the available space given by its parent, and set only vertical paddings